tests/conftest.py

```diff
@@ -9,11 +9,18 @@
 
 def pytest_addoption(parser: Parser):
     """Add options to pytest parser."""
+    # The path to repository in <org>/<repo> or <user>/<repo> format.
```

Comment:
Could these be help messages for the parser?

Reply:
The message is also added in the parser :D
```python3
path = pytestconfig.getoption("--path")
    assert path, (
        "Please specify the --path command line option with repository "
        "path of <org>/<repo> or <user>/<repo> format."
    )
```

Reply:
oh nvm i got what you mean

Reply:
Moved the comments as help messages, thank you!